boards/px4/fmu-v6c/default.px4board

```diff
@@ -10,7 +10,7 @@ CONFIG_DRIVERS_BAROMETER_MS5611=y
 CONFIG_DRIVERS_BATT_SMBUS=y
 CONFIG_DRIVERS_CAMERA_CAPTURE=y
 CONFIG_DRIVERS_CAMERA_TRIGGER=y
-CONFIG_COMMON_DIFFERENTIAL_PRESSURE=y
+CONFIG_DRIVERS_DIFFERENTIAL_PRESSURE_ASP5033DRIVER=y
```

Comment:
I don't think you can change the existing entry like that. It would have to be added to the build or just be optional for anyone adding this to the build.

So I think you need to add it to the Kconfig list in src/drivers/differential_pressure and have it as default off.